CodeGeneration/Sources/SyntaxSupport/Utils.swift

```diff
@@ -38,21 +38,39 @@ public func lowercaseFirstWord(name: String) -> String {
   return name.prefix(wordIndex).lowercased() + name[name.index(name.startIndex, offsetBy: wordIndex)..<name.endIndex]
 }
 
-/// Give a (possibly multi-line) string, prepends `///` to each line and creates
-/// a `.docLineComment` trivia piece for each line.
-public func docCommentTrivia(from string: String?) -> SwiftSyntax.Trivia {
-  guard let string else {
-    return []
-  }
-  let lines = string.split(separator: "\n", omittingEmptySubsequences: false)
-  let pieces = lines.enumerated().map { (index, line) in
-    var line = line
-    if index != lines.count - 1 {
-      line = "\(line)\n"
+// Helpers to create trivia pieces
+extension SwiftSyntax.Trivia {
```

Comment:
The reason I moved `docCommentTrivia` is just to be consistent, and to hint at where it is located.